components/gitpod-db/src/typeorm/team-db-impl.ts

```diff
@@ -15,6 +16,34 @@ import { DBTeamMembership } from "./entity/db-team-membership";
 import { DBUser } from "./entity/db-user";
 import { DBTeamMembershipInvite } from "./entity/db-team-membership-invite";
 
+const FORBIDDEN_SLUGS = [
+    'access-control',
+    'account',
+    'admin',
+    'blocked',
+    'from-referrer',
+    'install-github-app',
+    'integrations',
+    'login',
+    'new',
+    'notifications',
+    'oauth-approval',
+    'plans',
+    'preferences',
+    'projects',
+    'settings',
+    'setup',
+    'sorry',
+    'start',
+    'subscription',
+    'teams',
+    'upgrade-subscription',
+    'usage',
+    'variables',
+    'workspaces',
+    ...(blocklist),
```

Comment:
**suggestion**: What do you think of adding a few more slugs that could beuseful in the near future?

For example, we could include `snapshots`, `prebuilds`, as well as `branches`, `issues`, `pull-requests`, and `merge-requests`. Your call! 🏓